kubetest/main.go

```diff
@@ -112,14 +115,15 @@ type options struct {
 
 func defineFlags() *options {
 	o := options{}
-	flag.Var(&o.build, "build", "Rebuild k8s binaries, optionally forcing (release|quick|bazel) strategy")
+	flag.Var(&o.build, "build", "Rebuild k8s binaries, optionally forcing (release|quick|bazel|dind) strategy")
```

Comment:
looks `--build=dind` is not supported yet? drop this for now so user won't get confused(too excited)?

Reply:
It's supported. Staging the artifacts for remote testing is not. I have some more code that does that.